.devcontainer/cuda12.3-oneapi2023.2.0/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+{
+  "shutdownAction": "stopContainer",
+  "image": "rapidsai/devcontainers:23.12-cpp-oneapi2023.2.0-cuda12.3-ubuntu22.04",
+  "hostRequirements": {
+    "gpu": true
+  },
+  "initializeCommand": [
+    "/bin/bash",
+    "-c",
+    "mkdir -m 0755 -p ${localWorkspaceFolder}/.{aws,cache,config}"
+  ],
+  "containerEnv": {
+    "SCCACHE_REGION": "us-east-2",
+    "SCCACHE_BUCKET": "rapids-sccache-devs",
+    "VAULT_HOST": "https://vault.ops.k8s.rapids.ai",
+    "HISTFILE": "${containerWorkspaceFolder}/.cache/._bash_history",
+    "DEVCONTAINER_NAME": "cuda12.3-oneapi2023.2.0",
```

Comment:
Something here is fishy, in the error logs I find:
```
lit: /home/coder/cccl/libcudacxx/test/utils/libcudacxx/test/config.py:364: note: detected host_cxx.version as: (20.21, 2.1, 1)
```

Reply:
I think this is because 2021.10 is the last release of the "classic" compilers that is bundled with the OneAPI package.